local/php/envs.go

```diff
@@ -37,11 +37,30 @@ func (p *Server) generateEnv(req *http.Request) map[string]string {
 	}
 
 	pathInfo := req.URL.Path
+	filenameMatch := false
+
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
 			scriptName = file
 			pathInfo = pathInfo[pos+4:]
+			filenameMatch = true
+		}
+
+	}
+
+	if pathInfo != string(os.PathSeparator) && filenameMatch == false {
+		paths := strings.Split(strings.TrimRight(pathInfo[1:], string(os.PathSeparator)), string(os.PathSeparator))
+		pathsLength := len(paths)
+		for i := 0; i < pathsLength; i++ {
```

Comment:
the loop should probably be reversed so that an `index.php` higher in the tree does not take precedence (this case should probably be added to the tests)

Reply:
It is currently as I just use -i in removal from array, but good catch about the test. Need to add one more nested level to actually be sure.

Reply:
Also, I am gonna rewrite the loop using pop push as it's actually confusing as hell.

Reply:
Actually, golang pop syntax is actually more complicated than current one so no rewrite, but I am adding tests.